packages/grafana-ui/src/components/Select/types.ts

```diff
@@ -64,6 +64,12 @@ export interface SelectCommonProps<T> {
   /** Sets the width to a multiple of 8px. Should only be used with inline forms. Setting width of the container is preferred in other cases.*/
   width?: number;
   isOptionDisabled?: () => boolean;
+  /** allowCustomValue must be enabled. Determines whether the "create new" option should be displayed based on the current input value, select value and options array. */
+  isValidNewOption?: (
```

Comment:
Just a side note - it seems like most of props are sorted alphabetically. It'd be nice to put them back in order for readability :)